test/views/pages/existingProtections/existingProtectionsSpec.scala

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.pages.existingProtections
+
+import config.FrontendAppConfig
+import models.{ExistingProtectionDisplayModel, ExistingProtectionsDisplayModel}
+import org.jsoup.Jsoup
+import play.api.i18n.Messages
+import play.api.i18n.Messages.Implicits._
+import play.api.mvc.Call
+import testHelpers.ViewSpecHelpers.{CommonMessages, CommonViewSpecHelper}
+import testHelpers.ViewSpecHelpers.existingProtections.existingProtections
+import views.html.pages.existingProtections.{existingProtections => views}
+
+class existingProtectionsSpec extends CommonViewSpecHelper with existingProtections with CommonMessages {
```

Comment:
Existing protections has logic in the view based on the data passed in. In order to fully test this all combinations of data that require logic checking need to be tested as well (in this case, having both an active protection and empty active protection as well as a case for no other protections and for other protections.)